ostree-restore.sh

```diff
@@ -24,12 +24,10 @@ base_container=$(ostree cat backup /rpm-ostree.status | awk '/ostree-unverified-
 
 log_it Importing base OCI
 cp pull-secret.json /etc/ostree/auth.json
-ostree container unencapsulate --repo /ostree/repo $base_container | tee /tmp/unencapsulate.base
-ostree refs --create base $(cut -d ' ' -f 2 < /tmp/unencapsulate.base)
 
 ostree admin os-init --sysroot /sysroot $new_osname
 log_it Deploying new stateroot
-ostree admin deploy --retain --sysroot /sysroot --os $new_osname base
+ostree container image deploy --sysroot /sysroot --stateroot $new_osname --imgref ${base_container}
```

Comment:
This also wants `--sysroot /` per https://github.com/eranco74/sno-relocation-poc/pull/26

Reply:
This does not work for me. Has the same behaviour that when using `ostree admin deploy --sysroot /sysroot`

I wrote it down in https://github.com/ostreedev/ostree/issues/2992#issuecomment-1684142845

Reply:
For 4.12/rhel8 it may work to `mount --bind /boot /sysroot/boot` if we need that

Reply:
confirmed experimentally that this syntax works properly on 4.13+

Reply:
I've updated this PR